app/assets/javascripts/two-pane.js

```diff
@@ -28,6 +28,13 @@ $(document).on('click', "a[href$='new'].new_two_pane", function(e) {
   }
 });
 
+$(document).on('click', "a[href$='edit'].edit_two_pane", function(e) {
```

Comment:
I used the class `display-two-pane` and I added it to every place I found that used two-pane.